src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -340,11 +340,12 @@ configurationRegistry.registerConfiguration({
 		},
 		'explorer.openEditors.sortOrder': {
 			'type': 'string',
-			'enum': ['editorOrder', 'alphabetical'],
+			'enum': ['editorOrder', 'alphabetical', 'fullPath'],
 			'description': nls.localize({ key: 'openEditorsSortOrder', comment: ['Open is an adjective'] }, "Controls the sorting order of editors in the Open Editors pane."),
 			'enumDescriptions': [
 				nls.localize('sortOrder.editorOrder', 'Editors are ordered in the same order editor tabs are shown.'),
-				nls.localize('sortOrder.alphabetical', 'Editors are ordered in alphabetical order inside each editor group.')
+				nls.localize('sortOrder.alphabetical', 'Editors are ordered in alphabetical order inside each editor group.'),
```

Comment:
Please change this to `..alphabetical order by {...} inside...` to help explain the difference between this and the next option.